sdk/keyvault/arm-keyvault/tsconfig.json

```diff
@@ -14,6 +14,6 @@
     "outDir": "./esm",
     "importHelpers": true
   },
-  "include": ["./lib/**/*.ts"],
+  "include": ["./src/**/*.ts"],
```

Comment:
This needs to be reflected in the generator as well. CC @daschult.

Reply:
https://github.com/Azure/autorest.typescript/pull/439